ca/certificates.go

```diff
@@ -119,6 +119,13 @@ type RootCA struct {
 	// must correspond to the key in the local signer, if provided
 	Cert []byte
 
+	// Intermediates contains a bundle of PEM encoded certificates to append to any issued TLS certificates.
+	// If more than one are provided, the first one must have the same public key and subject as the signing
+	// root certificate (or, if no key is provided, at least one of the root certificates), and the rest
```

Comment:
No sorry, my comment was not clear - just that a chain can be formed using the intermediates to one of the root certs in the pool (e.g. the intermediates doesn't have to have the root, but one of them should be signed by one of the roots)